### PR TITLE
Update create-test-tune-dlp-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details about permissions to the page

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2538

File: microsoft-365/compliance/create-test-tune-dlp-policy.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ DLP uses a content analysis engine to examine the contents of email messages and
...
 
+## Permissions
+
+Members of your compliance team who will create DLP policies need permissions to the Security &amp; Compliance Center. By default, your tenant admin will have access to this location and can give compliance officers and other people access to the Security &amp; Compliance Center, without giving them all of the permissions of a tenant admin. To do this, we recommend that you:
+  
+1. Create a group in Microsoft 365 and add compliance officers to it.
+    
+2. Create a role group on the **Permissions** page of the Security &amp; Compliance Center. 
+
+3. While creating the role group, use the **Choose Roles** section to add the following role to the Role Group: **DLP Compliance Management**.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-test-tune-dlp-policy.md]:
```suggestion
3. While creating the role group, use the **Choose Roles** section to add the following role to the role group: **DLP Compliance Management**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Pull request merged**